--- origsrc/liquidsoap-0.9.3/src/stream/float_pcm_c.c	2010-09-04 14:51:17.000000000 -0500
+++ src/liquidsoap-0.9.3/src/stream/float_pcm_c.c	2010-10-17 22:33:46.774468400 -0500
@@ -38,7 +38,10 @@
 #include <limits.h>
 #include <stdint.h>
 #include "config.h"
+
+#ifndef __CYGWIN__
 static inline int16_t bswap_16 (int16_t x) { return ((((x) >> 8) & 0xff) | (((x) & 0xff) << 8)); }
+#endif
 
 #include <assert.h>
 #include <stdio.h>
