--- origsrc/liquidsoap-1.1.1/doc/Makefile	2013-05-08 18:40:52.000000000 -0500
+++ src/liquidsoap-1.1.1/doc/Makefile	2013-10-29 20:54:20.474504000 -0500
@@ -65,7 +65,7 @@ liqi/liqi: all-subdirs
 
 content/reference.txt: reference_to_liqi.pl ../src/liquidsoap ../scripts/utils.liq
 ifeq ($(PERL_DOM),yes)
-	CAML_LD_LIBRARY_PATH=../src:../src/plugins ../src/liquidsoap --dynamic-plugins-dir ../src/plugins \
+	CAML_LD_LIBRARY_PATH=../src:../src/plugins ../src/liquidsoap \
 	  --no-pervasives ../scripts/pervasives.liq --list-plugins-xml | \
 	  ./reference_to_liqi.pl > $(@)
 else
@@ -75,7 +75,7 @@ else
 endif
 
 content/settings.txt: ../src/liquidsoap
-	CAML_LD_LIBRARY_PATH=../src:../src/plugins ../src/liquidsoap --dynamic-plugins-dir ../src/plugins \
+	CAML_LD_LIBRARY_PATH=../src:../src/plugins ../src/liquidsoap \
 	--conf-descr-liqi > $(@)
 
 VERSION="`../src/liquidsoap --version | head -n 1`"
